ui/packages/catalog-ui-search/src/main/webapp/react-component/container/table-export/table-export.tsx

```diff
@@ -74,6 +74,38 @@ function getHiddenFields(): string[] {
     .get('columnHide')
 }
 
+function getHits(sources: Source[]): number {
+  return sources
+    .filter(source => source.id !== 'cache')
+    .reduce((hits, source) => (source.hits ? hits + source.hits : hits), 0)
+}
+
+function getExportCount(exportSize: string, selectionInterface: any): number {
+  const result = selectionInterface.getCurrentQuery().get('result')
+  return exportSize === 'all'
+    ? getHits(result.get('status').toJSON())
+    : result.get('results').length
+}
+
+function getSorts(selectionInterface: any) {
+  return selectionInterface.getCurrentQuery().get('sorts')
+}
+
+function getQueryCount(selectionInterface: any): number {
+  return selectionInterface.getCurrentQuery().get('count')
+}
+
+function getWarning(exportSize: string, selectionInterface: any): string {
+  const exportCount = getExportCount(exportSize, selectionInterface)
+  if (exportCount > 100) {
```

Comment:
❓ Should we make this value configurable?

Reply:
I don't think so. It seems unlikely that anyone would want to configure this.